### ISSUE TITLE
DeDRM for new PDF (issued 2023) not working

### ISSUE THREAD

**Issue by schaumermaldann** [apprenticeharper/DeDRM_tools#2240]
I've got an PDF ebook issued 2023 which is not properly handled by DeDRM. My Windows app PDF-XChange Editor says "Unknown/unsupported security handler".
![DeDRM](https://github.com/apprenticeharper/DeDRM_tools/assets/134861550/bb955ef1-aac5-4375-a873-20719ce98c70)

My Environment is:

Calibre: 6.14
DeDRM: 10.0.3

p.s. It's my first post on github, so please apologize if it is not correct

**Comment by ElleKayEm** [comment]
How did you download the book?  Is it Adobe DRM?

**Comment by schaumermaldann** [comment]
Yes it's Adobe DRM. The book i from a public library. I can provide the book, however, please via a separate channel. You can I pass it to you? it's 33 MB.

**Comment by ElleKayEm** [comment]
Sending the book won't help.  ADE used to download the book and calibre/DeDRM need to be on the same computer and user.

**Comment by schaumermaldann** [comment]
Hm, how can I help then? In order to download the book yourselves, you'd my
library account.

ElleKayEm ***@***.***> schrieb am Mi., 31. Mai 2023, 14:47:

**Comment by ElleKayEm** [comment]
You could post a log: https://github.com/noDRM/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by schaumermaldann** [comment]
Okay, I'll do that on friday

ElleKayEm ***@***.***> schrieb am Mi., 31. Mai 2023, 18:41:

**Comment by schaumermaldann** [comment]
[tmpxz8kwqhm.txt](https://github.com/apprenticeharper/DeDRM_tools/files/11635004/tmpxz8kwqhm.txt)
Here we go...

**Comment by ElleKayEm** [comment]
That's probably a bug.  @noDRM or someone else would need to take a look at it.

**Comment by schaumermaldann** [comment]
Is there anything I need to do to preceed?

**Comment by ElleKayEm** [comment]
Unfortunately there hasn't been any activity on this project in a while.  You could try emailing noDRM but I don't know if you'll get a response.  Email shown in their profile here is no_drm123@protonmail.com.

**Comment by schaumermaldann** [comment]
No answer to my mail :-(

**Comment by noDRM** [comment]
I didn't receive any mail from you, or it might have gotten lost. Looking at that log it looks like your key in the DeDRM config got corrupted somehow. Can you remove the key from the config then try to add the book again? The plugin should then hopefully autodiscover and add the key again.

**Comment by schaumermaldann** [comment]
Not sure if I tried the right thing: I went to config/extensions and removed the key for "Adobe Digital Edition ebook". Then I imported the corrupt PDF again. Still could not open it. Looking for the key it didn't reappear. Then I imported another ebook w/o trouble and the key reappeared. Then I reimporting the ebook in trouble again and it still didn't work. There are only rare ebooks having the problem.

**Comment by ElleKayEm** [comment]
Was the book that worked from the same source?  And downloaded with the same ADE?

**Comment by schaumermaldann** [comment]
Yes. Same source, same ADE. 99% of the book from this source work ok. PDFs and epubs.
